docker/judgehost/chroot-and-tar.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 
-# Add packages with -i "<apt package name>" here
+# /opt/domjudge/judgehost/bin/dj_make_chroot CLI options:
```

Comment:
I agree documentation is good, but isn't it smarter to link to the relevant section of the file (I think: https://github.com/DOMjudge/domjudge/blob/main/misc-tools/dj_make_chroot.in#L62-L81) ? This prevents problems if the other file is updated and we forget to update this file.